java/feature-layer-update-attributes/src/main/java/com/esri/arcgisruntime/sample/featurelayerupdateattributes/MainActivity.java

```diff
@@ -105,7 +104,7 @@ protected void onCreate(Bundle savedInstanceState) {
     map.getOperationalLayers().add(mFeatureLayer);
 
     // set an on touch listener to listen for click events
-    mMapView.setOnTouchListener(new DefaultMapViewOnTouchListener(this, mMapView) {
+    mapView.setOnTouchListener(new DefaultMapViewOnTouchListener(this, mapView) {
```

Comment:
As per other similar sample, lets hide the existing callout at the same time as the selection is cleared.